cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/type/DefaultApplicationTag.java

```diff
@@ -5,6 +5,7 @@
     OWNER("Owner"),
     CDP_USER_NAME("cdp-user-name"),
     CDP_CB_VERSION("cdp-cb-version"),
+    ENVIRONMENT_NAME("environment-name"),
```

Comment:
guess no since environment is a cloudbreak logical entity, it has nothing to do with the whole cdp, hence I didn't add this prefix to the actual value. but please correct me if I'm wrong.
@lacikaaa what is your opinion about it?